lib/theme/text_theme.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'colors.dart';
+
+/// The referenced design (Figma) is based on the Unity Style Typography. But
+/// the one we are supposed to follow is Material Design Typography. Each [TextStyle]
+/// defined below corresponds to the matching text style defined in Unity.
+const TextTheme textTheme = TextTheme(
```

Comment:
How can you handle the textTheme if tomorrow I come to you, ask for dark theme in the app?

Reply:
Here, for both light or dark theme of the app, if I'm not supposed to change any of the textStyle, except the textColor, I will use this: `textTheme.apply(display/bodyColor: <Some shade of a light color, probably white>)`. 
But if the point is to create to different textStyles, then I think I will have to define TextTheme object with a light color.